cloud-debugging-service/src/debug_session.rs

```diff
@@ -396,6 +399,21 @@ fn get_oplog_entry_from_public_oplog_entry(
         PublicOplogEntry::PendingUpdate(_) => {
             Err("Cannot override an oplog with a pending update".to_string())?
         }
+        PublicOplogEntry::BeginRemoteTransaction(_) => {
+            Err("Cannot override an oplog with a begin remote transaction".to_string())?
+        }
+        PublicOplogEntry::PreCommitRemoteTransaction(_) => {
+            Err("Cannot override an oplog with a pre commit remote transaction".to_string())?
+        }
+        PublicOplogEntry::CommittedRemoteTransaction(_) => {
+            Err("Cannot override an oplog with a committed remote transaction".to_string())?
+        }
+        PublicOplogEntry::PreRollbackRemoteTransaction(_) => {
+            Err("Cannot override an oplog with a pre rollback remote transaction".to_string())?
+        }
+        PublicOplogEntry::RolledBackRemoteTransaction(_) => {
```

Comment:
I might find the answer as I go the PR. But what's the guarantee of having `RolledBackRemoteTransaction` for every successful rollback?